test/nlp_solver.jl

```diff
@@ -40,7 +38,7 @@ new_optimizer() = IpoptOptimizer(print_level=0)
         #     1 <= x1, x2, x3, x4 <= 5
         # Start at (1,5,5,1)
         # End at (1.000..., 4.743..., 3.821..., 1.379...)
-        m = Model(optimizer=new_optimizer())
+        m = Model(with_optimizer(IpoptOptimize, print_level=0))
```

Comment:
Typo: IpoptOptimizer. This file isn't run under CI because it depends on Ipopt. Please test manually.